setup.py

```diff
@@ -8,6 +8,9 @@
 import os
 import sys
 
+from fmriprep.info import (__version__, __author__, __copyright__, __credits__,
```

Comment:
This should not be necessary, we have this: https://github.com/rwblair/fmriprep/blob/3866c4e555f4ebef0d1fc2dab0c4013126a60846/setup.py#L23 for that.

The reason is: if you import fmriprep in the setup.py, then you need to have fmriprep installed for setup.py to work correctly.